### ISSUE TITLE
A11y - radio field

### ISSUE THREAD

**Issue by anqaka** [SnowdogApps/magento2-alpaca-components#194]
Radio field accessibility issue - using keyboard user cannot uncheck radio option if once staring to iterate through option - to check

**Comment by bartdominiak** [comment]
In my opinion is working properly. You need to use arrows to switch the option and if you want to select current state u need to use space (regardes to https://stackoverflow.com/a/30233281)

And also you cannot unselect radio button after select, it's a normal behavior.

**Comment by anqaka** [comment]
👍
